README.md

```diff
@@ -667,7 +667,46 @@ make run-tests
 
 This will run a set of both unit `(TODO)` and integration tests. See the `Makefile` for more details
 
+<br />
+
+## Local packages development
+
+The [bento_web](https://github.com/bento-platform/bento_web) and 
+[bento_public](https://github.com/bento-platform/bento_public) projects use Bento specific packages for 
+[charts](https://github.com/bento-platform/bento_charts/) and [auth](https://github.com/bento-platform/bento_auth_js).
+
+In production, these packages are installed from the NPM registry.
+When developing simultanuously on a given web app and package, 
+it is useful to be able to use the locally built package whithout having to publish unfinished dev packages to NPM.
```

Comment:
typo: should be `without`

Reply:
also maybe wrap this line to be a consistent line length alongside others? or just re-flow this in general to have consistent line lengths in the markdown